admin_guide/managing_networking.adoc

```diff
@@ -378,6 +418,227 @@ If only some of the nodes in your cluster are capable of claiming the specified
+endif::openshift-enterprise[]
+ifdef::openshift-origin[]
+    image: openshift/origin-pod
+endif::openshift-origin[]
+----
+<1> This uses the YAML syntax for a multi-line string; see below for
+details.
+====
+
+Each line of `EGRESS_DESTINATION` can be one of three types:
+
+- `<port> <protocol> <IP address>` - This says that incoming
+connections to the given `<port>` should be redirected to the same
+port on the given `<IP address>`. `<protocol>` is either `tcp` or
+`udp`. In the example above, the first line redirects traffic from
+local port 80 to port 80 on 203.0.113.25.
+- `<port> <protocol> <IP address> <remote port>` - As above, except
+that the connection is redirected to a different `<remote port>` on
+`<IP address>`. In the example above, the second and third lines
+redirect local ports 8080 and 8443 to remote ports 80 and 443 on
+203.0.113.26.
+- `<fallback IP address>` - If the last line of `EGRESS_DESTINATION`
+is a single IP address, then any connections on any other port will be
+redirected to the corresponding port on that IP address (eg,
+203.0.113.27 in the example above). If there is no fallback IP address
+then connections on other ports would simply be rejected.)
+
+[[admin-guide-manage-pods-egress-router-configmap]]
+==== Using a ConfigMap to specify EGRESS_DESTINATION
+
+In the case of a large or frequently-changing set of destination
+mappings, it may be useful to maintain the list externally in a
+ConfigMap, and have the egress router pod read it from there. This
+also has the advantage that project administrators would be able to
+edit the ConfigMap (whereas they may not be able to edit the Pod
+definition directly, since it contains a privileged container).
+
+To do this:
+
+. Create a file containing the `EGRESS_DESTINATION` data:
++
+----
+$ cat my-egress-destination.txt
+# Egress routes for Project "Test", version 3
+
+80   tcp 203.0.113.25
+
+8080 tcp 203.0.113.26 80
+8443 tcp 203.0.113.26 443
+
+# Fallback
+203.0.113.27
+----
++
+Note that you can put blank lines and comments into this file
+
+. Create a ConfigMap object from the file:
++
+----
+$ oc delete configmap egress-routes --ignore-not-found
+$ oc create configmap egress-routes --from-file=destination=my-egress-destination.txt
+----
++
+Here `egress-routes` is the name of the ConfigMap object being
+created and `my-egress-destination.txt` is the name of the file the
+data is being read from.
+
+. Create a egress router pod definition as above, but specifying the
+ConfigMap for `EGRESS_DESTINATION` in the environment section:
++
+----
+    ...
+    env:
+    - name: EGRESS_SOURCE
+      value: 192.168.12.99
+    - name: EGRESS_GATEWAY
+      value: 192.168.12.1
+    - name: EGRESS_DESTINATION
+      valueFrom:
+        configMapKeyRef:
+          name: egress-routes
+          key: destination
+    - name: EGRESS_ROUTER_MODE
+      value: init
+    ...
+----
+
+Note that the egress router will not automatically update when the
```

Comment:
@derekwaynecarr @bparees not sure who to ask on this but is this a bug (or do I just not understand how configmap and DC/RC's work)? Shouldn't DC/RC's be update with config maps referenced by them change?